doc/whats_new.rst

```diff
@@ -110,6 +110,8 @@ BUG
 
         - Fix comparisons of filter settings for determining "strictest" filter
 
+    - Fixes to plotting and timing of :class:`mne.Annotations <annotations>` and restricted addition of annotations outside data range to prevent problems with cropping and concatenating data by `Jaakko Leppakangas`_
```

Comment:
And check the rendering, I think you mean:
```
:class:`annotation <mne.Annotations>`
```
but really this is good enough:
```
:class:`Annotations`
```
We're already in the `mne` namespace in this file